lib/MongoDB/Op/_BulkWrite.pm

```diff
@@ -74,6 +87,46 @@ sub has_collation {
     } @{ $self->queue };
 }
 
+## Encapsulate the write loop for this
```

Comment:
Please figure out what of this comment needs to be kept (if any) for documentation purposes and what can be deleted.